test/fixtures/cookbooks/zabbix_lwrp_test/recipes/zabbix24.rb

```diff
@@ -2,6 +2,10 @@
 include_recipe 'postgresql_lwrp::apt_official_repository'
 include_recipe 'nginx::official-repo'
 
+# Temporary use higher version of zabbixapi, for correct tests works
+# In gem zabbixapi==2.4.X uses old json gem (==1.6.1) but in chefdk uses newest version
+node.default['zabbix']['api-version'] = '3.0.0'
```

Comment:
Are zabbix24.rb and +node.default['zabbix']['api-version'] = '3.0.0' changes inserted as planned?